### ISSUE TITLE
Facilitating review of data badges and also minimal requirements

### ISSUE THREAD

**Issue by nheeren** [majeau-bettez/opendata_jie#9]
When reviewing the compliance of submissions for the data badges it is often very cumbersome to figure out where the data is located.

Therefore, I suggest we come up with a template spreadsheet that authors requesting a data badge have to fill out. An additional benefit would be that this "barrier" makes it clear to authors the "I request a data badge" checkbox should not be clicked for fun (only).

Examples: 
- _Please summarise the system description in the following field and mention where this is found in the manuscript_:  `xxx`
- _Please indicate if this is a gold or silver contribution and why._: `xxx`

Having such a template for the minimum requirements would be great too, but I feel like we will be causing too much hassle.

**Comment by rlifset** [comment]
Guillaume and Niko,

We (in the editorial office) are looking at all the logistical issues related to data requirements/badges. This suggestion is a good one, but we need to figure out how best to make it work in ScholarOne.

Thanks,
Reid

**Comment by nheeren** [comment]
Sure we need to find a solution that works in S1. But wouldn't it be possible to ask authors to include a e.g. xlsx file into their submission? 

How can we make progress on this? 
What is Guillaume's opinion?

**Comment by rlifset** [comment]
Yes, but we need to decide how to do this, e.g., an email after submission (when an author has requested a badge)?  During submission (i.e., within S1M)?

There is an annoying amount of detail involved if a useful routine is to be established.  (The real issue is that other people in the editorial office are working furiously on getting new (slow) staff up to speed.)

**Comment by nheeren** [comment]
Maybe @majeau-bettez and I should already draft a template / questions that we deem useful and then we can see how to get this into the S1 processes?
